src/components/ShowCaseChampionsList.vue

```diff
@@ -40,9 +40,11 @@ export default {
     ...mapGetters(["champions"]),
     filteredChampions() {
       return this.champions.filter(champion =>
-        champion.name
-          .toLowerCase()
-          .includes(this.championsCriteria.toLowerCase())
+        this.championsCriteria.length > 2
+          ? champion.name
+              .toLowerCase()
+              .includes(this.championsCriteria.toLowerCase())
+          : champion.name
```

Comment:
1. A thing i've come to realize some days ago, you don't need to use `String.Prototype.toLowerCase()` or `String.Prototype.toUpperCase()`, it is not 100% safe. Besides you should use `localeCompare()` which is a `case-insensitive` comparison but it's `accent-sensitive`.
```js
...champion.name.localeCompare(this.championsCriteria)
```
2. Also, i do think this code is kinda convoluted and you could make something like
```js
filteredChampions () {
  const {championsCriteria, champions} = this
  if (championsCriteria.length <= 2) return champions
  return champions.filter(({ name }) => champion.name.localeCompare(championsCriteria)
}
```

Reply:
Done!